src/components/PolicyholdersView/PolicyholdersView.tsx

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios';
+import { Box, Button } from '@mui/material';
+import { useEffect, useState } from 'react';
+
+import InfoTable from 'components/InfoTable';
+import { fakeData } from 'constants/data';
+import { PolicyholdersType } from 'types';
+
+type PolicyholdersResponseType = {
+  policyHolders: PolicyholdersType[];
+};
+
+function PolicyholdersView() {
+  const [policyholders, setPolicyholders] = useState<
+    PolicyholdersType[] | undefined
+  >();
+
+  // TODO: handle error and loading states in both fns below.
+  const getPolicyholders = async () => {
+    try {
+      let { data } = await axios.get<PolicyholdersResponseType>(
+        'https://fe-interview-technical-challenge-api-git-main-sure.vercel.app/api/policyholders'
+      );
+      setPolicyholders(data.policyHolders);
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  const createPolicyholder = async () => {
+    try {
+      let { data } = await axios.post<PolicyholdersResponseType>(
+        'https://fe-interview-technical-challenge-api-git-main-sure.vercel.app/api/policyholders',
+        fakeData
+      );
+      setPolicyholders(data.policyHolders);
+    } catch (e) {
+      console.log(e);
+    }
+  };
```

Comment:
Ideally in a real app, when we make a POST request it would save the data in our database so the next time we do a GET or POST, the previous results would also be returned. If for some reason that couldn't be the case but we wanted to have good UX, we could use local storage to save the returned data so we could show the previously created tables to the user each time they revisit the page.
However, for the purpose of this challenge, since we're hardcoding the data and not collecting any data from the user, it's unnecessary.